tools/src/scie_pants/pants_version.py

```diff
@@ -18,19 +18,27 @@
 from packaging.specifiers import SpecifierSet
 from packaging.version import Version
 
-from scie_pants.log import info, warn
+from scie_pants.log import fatal, info, warn
 from scie_pants.ptex import Ptex
 
 log = logging.getLogger(__name__)
 
+# After this version, Pants is released as a per-platform PEX using GitHub Release assets.
+# See https://github.com/pantsbuild/pants/pull/19450
+PANTS_PEX_GITHUB_RELEASE_VERSION = Version("2.18.0.dev3")
```

Comment:
This'll need to be adjusted to the actual right version for when the PEXs were released.
Additionally, the comment should reference the commit on `main`.

(We could technically backport this onto older releases and adjust this version downward. But that's a separate discussion)